src/main/kotlin/com/atlassian/performance/tools/report/api/result/Stats.kt

```diff
@@ -0,0 +1,14 @@
+package com.atlassian.performance.tools.report.api.result
+
+import java.time.Duration
+
+/**
+ * TODO: Document this file here
```

Comment:
TODO: Document this file here

Reply:
https://en.wikipedia.org/wiki/Statistical_parameter :
> Parameters of this type are given names appropriate to their roles, including the following.
> * location parameter
> * dispersion parameter or scale parameter
> * shape parameter

It contains links explaining location and dispersion/scale.